### ISSUE TITLE
mcrun failed with Error `No such file or directory`

### ISSUE THREAD

**Issue by stc1988** [Moddable-OpenSource/moddable#1408]
**Build environment:** macOS,
**Moddable SDK version:**  a9e80d0
**Target device:** esp32/m5stack, esp32/m5stack_core2, esp32/moddable_six

**Steps to Reproduce**
1. Build and install the app `examples/piu/outline/figures`  using this build command: `mcconfig -d -m -p esp32/m5stack`
2. Install the mod `CanvasPath` using `mcrun -d -m -p esp32/m5stack`

```
mcrun -d -m -p esp32/m5stack
### Error: No such file or directory
```

**Comment by phoddie** [comment]
Thanks for the report. It works fine in the simulator. It looks like a side-effect of recent changes to support ESP Components. We'll take a look.

**Comment by mkellner** [comment]
Hello @stc1988, I've committed a fix. Please fetch the latest and rebuild tools.

**Comment by stc1988** [comment]
`mcrun` is working now, but `mcconfig` has stopped working.

```
mcconfig -d -m -p esp32/m5stack_core2
mcconfig: mod partition of size 0x40000 created from factory app partition
/Users/satoshi/.local/share/esp32/esp-idf-v5.3/tools/idf.py
/Users/satoshi/Projects/moddable/build/tmp/esp32/m5stack_core2/debug/figures/makefile:1526: warning: overriding commands for target `/Users/satoshi/Projects/moddable/build/tmp/esp32/m5stack_core2/debug/figures/ftimage.h.xsi'
/Users/satoshi/Projects/moddable/build/tmp/esp32/m5stack_core2/debug/figures/makefile:1457: warning: ignoring old commands for target `/Users/satoshi/Projects/moddable/build/tmp/esp32/m5stack_core2/debug/figures/ftimage.h.xsi'
make: *** No rule to make target `/Users/satoshi/Projects/moddable/build/tmp/esp32/m5stack_core2/debug/figures/xsProj-esp32/../xs_idf_deps.txt', needed by `dependencies'.  Stop.
make: *** Waiting for unfinished jobs....
rm: /Users/satoshi/Projects/moddable/build/tmp/esp32/m5stack_core2/debug/figures/xsProj-esp32/main/idf_component.yml: No such file or directory
Using recommended ESP-IDF v5.3
```

**Comment by mkellner** [comment]
My apologies. Please try again?

**Comment by stc1988** [comment]
I confirmed it fixed. Thank you!!

**Comment by stc1988** [comment]
@mkellner Since this issue occurs when installing Moddable using `xs-dev`, would it be possible to release version `5.0.1`?

**Comment by mkellner** [comment]
I have released 5.0.1

**Comment by stc1988** [comment]
Thank you for responding so quickly!
